docs/docsite/requirements.txt

```diff
@@ -1,13 +1,18 @@
-#pip packages required to build docsite
+# pip packages required to build docsite
+# these requirements are as loosely defined as possible
+# if you want known good versions of these dependencies
+# use known_good_reqs.txt instead 
+
+antsibull >= 0.25.0
+docutils == 0.16 # pin for now until the problem with unordered lists is fixed
+# see https://github.com/readthedocs/sphinx_rtd_theme/issues/1115jinja2==3.0.1
 jinja2
+Pygments >= 2.4.0
```

Comment:
This is not necessary among the direct deps anymore because the theme pulls in this dep and we no longer use it directly.
```suggestion
```